docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+
+swarm.elytron.sasl-authentication-factories._KEY_.security-domain:: 
+The SecurityDomain to associate with this resource
+
+swarm.elytron.security-domains._KEY_.default-realm:: 
+The default realm contained by this security domain.
+
+swarm.elytron.security-domains._KEY_.outflow-anonymous:: 
+When outflowing to a security domain if outflow is not possible should the anonymous identity be used?  Outflowing anonymous has the effect of clearing any identity already established for that domain.
+
+swarm.elytron.security-domains._KEY_.outflow-security-domains:: 
+The list of security domains that the security identity from this domain should automatically outflow to.
+
+swarm.elytron.security-domains._KEY_.permission-mapper:: 
+A reference to a PermissionMapper to be used by this domain.
+
+swarm.elytron.security-domains._KEY_.post-realm-principal-transformer:: 
+A reference to a principal transformer to be applied after the realm has operated on the supplied identity name.
+
+swarm.elytron.security-domains._KEY_.pre-realm-principal-transformer:: 
+A reference to a principal transformer to be applied before the realm is selected.
+
+swarm.elytron.security-domains._KEY_.principal-decoder:: 
+A reference to a PrincipalDecoder to be used by this domain.
+
+swarm.elytron.security-domains._KEY_.realm-mapper:: 
+Reference to the RealmMapper to be used by this domain.
+
+swarm.elytron.security-domains._KEY_.realms:: 
+The list of realms contained by this security domain.
+
+swarm.elytron.security-domains._KEY_.role-mapper:: 
+Reference to the RoleMapper to be used by this domain.
+
+swarm.elytron.security-domains._KEY_.security-event-listener:: 
+Reference to a listener for security events.
+
+swarm.elytron.security-domains._KEY_.trusted-security-domains:: 
+The list of security domains that are trusted by this security domain.
+
+swarm.elytron.security-properties:: 
+Security properties to be set.
+
+swarm.elytron.server-ssl-contexts._KEY_.active-session-count:: 
+The count of current active sessions.
+
+swarm.elytron.server-ssl-contexts._KEY_.authentication-optional:: 
+Rejecting of the client certificate by the security domain will not prevent the connection. Allows a fall through to use other authentication mechanisms (like form login) when the client certificate is rejected by security domain. Has an effect only when the security domain is set.
+
+swarm.elytron.server-ssl-contexts._KEY_.cipher-suite-filter:: 
+The filter to apply to specify the enabled cipher suites.
+
+swarm.elytron.server-ssl-contexts._KEY_.final-principal-transformer:: 
+A final principal transformer to apply for this mechanism realm.
+
+swarm.elytron.server-ssl-contexts._KEY_.key-manager:: 
+Reference to the key manager to use within the SSLContext.
+
+swarm.elytron.server-ssl-contexts._KEY_.maximum-session-cache-size:: 
+The maximum number of SSL sessions in the cache. The default value -1 means use the JVM default value. Value zero means there is no limit.
```

Comment:
Clean up value references here to use backticks and be consistent:
> `-1` means .... Value `0` means...

Reply:
Note to self: not easily fixable in our sources.